kinto_http/__init__.py

```diff
@@ -61,12 +60,25 @@ def get(self, endpoint, **kwargs):
                                  field=','.join(e.args)))
 
 
+class BearerTokenAuth(requests.auth.AuthBase):
+    def __init__(self, token):
+        self.token = token
+
+    def __call__(self, r):
+        r.headers["Authorization"] = "Bearer " + self.token
```

Comment:
This default is a bit tricky since we can change it on the server, we should be able to change it here as well, I guess we can use the auth parameter for that and access_token is just a helper, But why not use auth directly?